src/zerobouncesdk/zb_guess_format_response.py

```diff
@@ -0,0 +1,58 @@
+from typing import List
```

Comment:
please separate import groups with newlines accordingly: newline only after this standard import = before local imports,  and remove other newlines that are between local imports
also please keep only one type of import path (relative or absolute) for local imports